tests/test_recharge_pagination.py

```diff
@@ -0,0 +1,56 @@
+"""
+Test tap pagination of streams
+"""
+
+from tap_tester import runner, connections
+
+from base import RechargeBaseTest
+
+class RechargePaginationTest(RechargeBaseTest):
+
+    @staticmethod
+    def name():
+        return "tap_tester_recharge_pagination_test"
+
+    def test_run(self):
+        page_size = 250
+        conn_id = connections.ensure_connection(self)
+
+        # Checking pagination for streams with enough data
+        expected_streams = ["metafields_subscription", "onetimes"]
```

Comment:
Is it possible to generate more records for the other streams in order to get them under test?

Reply:
I've generated additional data for a couple more streams and added them to the test.

Adding data to the other streams might be a hassle as it has to be done through the attached Shopify store. Currently the streams look like so:
```bash
Details by stream:
+-------------------------+---------+---------+
| stream                  | records | schemas |
+-------------------------+---------+---------+
| addresses               | 258     | 1       |
| charges                 | 19      | 1       |
| collections             | 1       | 1       |
| customers               | 261     | 1       |
| discounts               | 367     | 1       |
| metafields_store        | 50      | 1       |
| metafields_customer     | 4       | 1       |
| metafields_subscription | 297     | 1       |
| onetimes                | 255     | 1       |
| orders                  | 12      | 1       |
| products                | 5       | 1       |
| shop                    | 1       | 1       |
| subscriptions           | 6       | 1       |
+-------------------------+---------+---------+
```